hades/models/validate.py

```diff
@@ -1,20 +1,27 @@
-from typing import Union
+from mongoengine import ValidationError, StringField, IntField
 
-from hades import db
 
+class EventMixin:
+    """
+    Class with common fields and validation
+    """
 
-class ValidateMixin(object):
-    def validate(self) -> Union[str, bool]:
-        table = self.__class__
-        # Ensure nobody else in the table has the same email address
-        if self.query.filter(table.email == self.email).first():
-            return f'Email address {self.email} already found in database! Please re-enter the form correctly!'
-
-        # Ensure nobody else in the table has the same phone number
-        for num in self.phone.split('|'):
+    def __validate_phone__(phone: str):
+        """
+        Ensure nobody else in the table has the same phone number
+        :param phone: phone number to be checked
+        """
+        for num in phone.split('|'):
             if len(str(num)) < 10:
-                return f'Phone number {num} is too short! Please re-enter the form correctly!'
-            if self.query.filter(table.phone.like(f'%{num}%')).first():
-                return f'Phone number {num} already found in database! Please re-enter the form correctly!'
+                raise ValidationError(
```

Comment:
Yeah so basically I wanna add the `validate()` back here to return plain old strings to the user to display as an error message